diffkemp/simpll/DebugInfo.cpp

```diff
@@ -412,29 +410,31 @@ void DebugInfo::collectMacrosWithValue(const Constant *Val) {
 
 /// Find all local variables and create a map from their names to their
 /// values.
-void DebugInfo::collectLocalVariables(std::set<const Function *> &Called,
+void DebugInfo::collectLocalVariables(
+        std::set<const Function *> &Called,
         std::unordered_map<std::string, const Value *> &Map) {
     for (auto Fun : Called) {
         for (auto &BB : *Fun) {
             for (auto &Inst : BB) {
                 if (!isa<CallInst>(Inst))
                     continue;
                 auto CInst = dyn_cast<CallInst>(&Inst);
-                if (!getCalledFunction(CInst->getCalledValue())->getName().
-                        startswith("llvm.dbg"))
+                if (!getCalledFunction(CInst->getCalledValue())
+                             ->getName()
+                             .startswith("llvm.dbg"))
                     continue;
 
                 // Get the value name and the value itself.
                 auto WrappedVal = CInst->getOperand(0);
-                auto ValMD = dyn_cast<MetadataAsValue>(WrappedVal)->
-                        getMetadata();
+                auto ValMD =
+                        dyn_cast<MetadataAsValue>(WrappedVal)->getMetadata();
                 auto Val = dyn_cast<ValueAsMetadata>(ValMD);
                 if (!Val)
                     continue;
 #if LLVM_VERSION_MAJOR < 6
                 MetadataAsValue *DIVal;
                 if (getCalledFunction(CInst->getCalledValue())->getName() ==
-                        "llvm.dbg.declare")
+                    "llvm.dbg.declare")
```

Comment:
I don't like the fact that this is aligned to the parenthesis since the last line of the condition has the same indent as the first command of the block and it is not clear where the split is. However, as far as I remember, this cannot be configured using `clang-format`, right?

Reply:
Technically, this is possible, but I do not recommend such an approach. It requires the disabling of this alignment after all `(` braces. Because of that even functions with a lot of parameters would end up looking like this (taken directly from `clang-format-8` documentation):
```
someLongFunction(argument1,
    argument2);
```
Instead, I suggest to break before the binary operands, so the break is more clearly visible as the line will begin with `&&`, `||` or `==`. I will implement this approach including what I have commented below so that the changes can be reviewed.

Reply:
This is a good solution, I like the way it looks in the code.